### ISSUE TITLE
Is override of view/catalog/show still needed for Spotlight v3.0?

### ISSUE THREAD

**Issue by elrayle** [cul-it/exhibits-library-cornell-edu#456]
Code location: `app/views/catalog/show.html.erb`

### Override from Spotlight v2.13.0

Original [Blacklight v6.3.3](https://github.com/projectblacklight/blacklight/blob/v6.3.3/app/views/catalog/show.html.erb) code:
```
<div id="content" class="<%= show_content_classes %>">
  <%= render_document_main_content_partial %>
</div>

<div id="sidebar" class="<%= show_sidebar_classes %>">
   <%= render_document_sidebar_partial %>
</div>
```

Override code:
```
<div id="content" class="col-sm-12">
  <%= render_document_main_content_partial %>
</div>
```

Purpose of original override was to not show the sidebar and to use a different class for the main content.  It is not clear why the main content class was not set with `show_content_classes`.

### Upgrade to Spotlight v3.0.0

Uses Blacklight v7.16.0 which is significantly different.  Need to reconcile.

[Blacklight v7.16.0](https://github.com/projectblacklight/blacklight/blob/v7.16.0/app/views/catalog/show.html.e) code:
```
<% if current_search_session %>
  <div id="appliedParams" class="clearfix constraints-container">
    <%= render 'start_over' %>
    <%= link_back_to_catalog class: 'btn btn-outline-secondary' %>
  </div>
<% end %>
<%= render_document_main_content_partial %>
<% content_for(:sidebar) do %>
  <%= render_document_sidebar_partial @document %>
<% end %>
```

Questions:
* Does the sidebar still need to be removed?
* Should any override of main content area be moved to `_render_document_main_content_partial`?

**Comment by mhk33** [comment]
I am looking at this as well and I think we probably don't need our own show.html.erb. Removing it would also fix #428. I am trying to test removing it locally, although I can't create a local exhibit at the moment (getting a solr error).

**Comment by mhk33** [comment]
I removed show.html.erb from the app and this is in dev now, and deployed to integration. All seems fine!
https://exhibits-integration.xcvhmd2mim.us-east-1.elasticbeanstalk.com/vintage-vision/catalog/5-173
